python/lsst/daf/butler/registry/connectionString.py

```diff
@@ -47,9 +47,8 @@ class ConnectionStringFactory:
 
     @classmethod
     def fromConfig(cls, registryConfig):
-        """Parses the 'db' key in the config, and if they exist username,
-        password, host, port and database keys, and returns an connection
-        string object.
+        """Parses the 'db' and, if they exist, username, password, host, port
```

Comment:
Maybe make the quoting consistent?  Add a comma before "and".  Add "the" before "given".  Maybe say that the `db` key is what specifies the initial/default connection string?

Reply:
Added a comment that `db` specifies the default connection string and is returned unchanged when no additional information about it is provided. Thanks!